projects/packages/sync/src/modules/class-search.php

```diff
@@ -0,0 +1,1793 @@
+		'_EventStartDate'                            => array(),
+		'_EventStartDateUTC'                         => array(),
+		'_EventTimezone'                             => array(),
+		'_EventTimezoneAbbr'                         => array(),
+		'_EventURL'                                  => array(),
+		'_EventVenueID'                              => array(),
+		'_OrganizerEmail'                            => array(),
+		'_OrganizerOrganizer'                        => array(),
+		'_OrganizerOrigin'                           => array(),
+		'_OrganizerPhone'                            => array(),
+		'_OrganizerWebsite'                          => array(),
+		'_VenueAddress'                              => array(),
+		'_VenueCity'                                 => array(),
+		'_VenueCountry'                              => array(),
+		'_VenueOrigin'                               => array(),
+		'_VenuePhone'                                => array(),
+		'_VenueProvince'                             => array(),
+		'_VenueShowMap'                              => array(),
+		'_VenueShowMapLink'                          => array(),
+		'_VenueState'                                => array(),
+		'_VenueStateProvince'                        => array(),
+		'_VenueURL'                                  => array(),
+		'_VenueVenue'                                => array(),
+		'_VenueVenueID'                              => array(),
+		'_VenueZip'                                  => array(),
+		'_default_attributes'                        => array(),
+		'_description'                               => array(),
+		'_edit_last'                                 => array(),
+		'_feedback_all_fields'                       => array(),
+		'_feedback_author'                           => array(),
+		'_feedback_author_email'                     => array(),
+		'_feedback_author_url'                       => array(),
+		'_feedback_contact_form_url'                 => array(),
+		'_feedback_ip'                               => array(),
+		'_feedback_subject'                          => array(),
+		'_file_paths'                                => array(),
+		'_layout'                                    => array(),
+		'_links_to'                                  => array(),
+		'_links_to_target'                           => array(),
+		'_mail'                                      => array(),
+		'_mail_2'                                    => array(),
+		'_messages'                                  => array(),
+		'_numero'                                    => array(),
+		'_post_restored_from'                        => array(),
+		'_video_url'                                 => array(),
+		'_website'                                   => array(),
+
+	); // end unindexed post meta.
+
+	/**
+	 * List of indexed taxonomy slugs - VARCHAR(32)
+	 *
+	 * @access private
+	 * @static
+	 *
+	 * @var array
+	 */
+	private static $taxonomies_to_sync = array(
```

Comment:
@gibrown -> can you outline what the purpose of this is?  In Sync the taxonomies sent to WP.com are determined by a blacklist [$blacklisted_taxonomies](https://github.com/Automattic/jetpack/blob/7d0fcdfc99c9fc2a1d1ddf60858c0a292cbdf385/projects/packages/sync/src/class-defaults.php#L415). Why are we maintaining a manual list of taxonomies that don't control actual data flows?

Ooo I get it this isn't for Jetpack's usage but instead is a whitelist on the WP.com side when creating the elastic doc object.  

If that is correct I might suggest a unit test that no values of $taxonomies_to_sync are contained in $blacklisted_taxonomies